www/views/customAmount.html

```diff
@@ -21,7 +21,7 @@
   <ion-content scroll="false">
     <div class="address" ng-if="address && amountBtc">
       <div class="qr-code" copy-to-clipboard="copyToClipboard()">
-        <qrcode size="220" data="bitcoin:{{address + '?amount=' + amountBtc + '&coin=' + coin}}" color="#334"></qrcode>
+        <qrcode size="220" data="bitcoin{{ wallet.coin == 'bch' ? 'cash' : '' }}:{{address + '?amount=' + amountBtc}}" color="#334"></qrcode>
```

Comment:
I think it would be better to have a fiunciton in walletService:

function getProtocolHandler(wallet) {
  if wallet.coin== 'bch' return 'bitcoincash';
  if wallet.coin=='btc' return 'bitcoin';
  <error>
}

and pass that info to the view, and do not put 'bitcoincash' in any other place.

Reply:
Updated. Thanks.